tests/test_update_metrics.py

```diff
@@ -0,0 +1,41 @@
+import csv
+import os
+import shutil
+import tempfile
+import unittest
+import src.tasks.tasks as tasks
+import torch
+
+
+class TestUpdateMetricsAccuracy(unittest.TestCase):
+    def setUp(self):
```

Comment:
we can't use snake case here? :(

Reply:
unittest has a specific namespace for testing,so setUp and tearDown has to use camel case in order to be run before and right after every test.